packages/xsnap/test/xsnap.test.js

```diff
@@ -489,7 +490,7 @@ test('GC after snapshot vs restore', async t => {
   globalThis.runToGC = (${runToGC});
   runToGC();
   // bloat the heap
-  send(Array.from(Array(2_000_000).keys()).length)
+  send(Array.from(Array(2_047_838).keys()).length);
```

Comment:
I binary searched this number down in hopes that it would be meaningful. It’s deterministic but the figure isn’t illuminating. Small changes project small changes to number that causes a failure here.